address-space-controller/src/main/resources/templates/standard-space-infra.yaml

```diff
@@ -462,8 +462,6 @@ objects:
             value: ${BROKER_PLUGIN_IMAGE}
           - name: TOPIC_FORWARDER_IMAGE
             value: ${TOPIC_FORWARDER_IMAGE}
-          - name: ${IMAGE_PULL_POLICY}
```

Comment:
I think the defect is that "- name: ${IMAGE_PULL_POLICY}" should read "- name: IMAGE_PULL_POLICY".  The purpose is to propagate the IMAGE_PULL_POLICY down to the resources created from the templates.